proxy/proxy.go

```diff
@@ -791,8 +791,9 @@ func WithParams(p Params) *Proxy {
 		defaultHTTPStatus = p.DefaultHTTPStatus
 	}
 
+	logger := &logging.DefaultLog{}
```

Comment:
I am not sure if we should use this, maybe just use logrus logger.
I think we do not use it consistently.